### PR TITLE
Add missing tutorials

### LINKED ISSUE #198: [TODO] Adjust  docs after `ProxyParser` PR is merged
- [x] Open TODOs in existing docs
- [x] Add #187 
- [x] ~~Add class graph for sitemaps~~
- [x] Add documentation for #182 
- [x] Add documentation for #183

### PR DESCRIPTION
**Pull request by MaxDall**
This adds missing tutorials for the library, updates the README.md, and adds some more logic to the filter behavior. 

The tutorials are:
1. Getting started
2. The Article class
3. How to filter articles
4. How to search for publishers

The changes made to the filter logic include.
- adding logic operators (or, and) to combine filters
**_NOTE:_** I'm not fully satisfied with how it is right now, but I'm fine with it. It still seems a bit complicated, but writing tutorial number three made it very obvious that we need something to combine filters. 

closes #198

### CODE REVIEW

File: docs/1_getting_started.md
Comment (Weyaaron) [inline docs/1_getting_started.md:4]:
```suggestion
- `PublisherCollection`
```

File: docs/1_getting_started.md
Comment (Weyaaron) [inline docs/1_getting_started.md:5]:
```suggestion
- `Crawler`
```

File: docs/1_getting_started.md
Comment (Weyaaron) [inline docs/1_getting_started.md:10]:
```suggestion
Fundus includes a collection of publisher-specific parsers grouped by country of origin.
```

File: docs/1_getting_started.md
Comment (Weyaaron) [inline docs/1_getting_started.md:12]:
```suggestion
```

File: docs/1_getting_started.md
Comment (Weyaaron) [inline docs/1_getting_started.md:14]:
```suggestion
To access all publishers based in the US:
```

File: docs/1_getting_started.md
Comment (Weyaaron) [inline docs/1_getting_started.md:23]:
```suggestion
If you want to crawl articles, the first step is to initialize a crawler.
```

File: docs/1_getting_started.md
Comment (Weyaaron) [inline docs/1_getting_started.md:26]:
```suggestion
Let's initiate a crawler capable of crawling publishers based in the US by utilizing
the `PublisherCollection`
```

**Review by Weyaaron — CHANGES_REQUESTED** [review]
I will stop adding individual changes to GitHub, instead I will open a PR from which @MaxDall may pick and choose changes. I will approve manually once the merge of the branches has taken place.

File: docs/3_how_to_filter_articles.md
Comment (Weyaaron) [inline docs/3_how_to_filter_articles.md:151]: Maybe all of these "überleitungen" should be reworked to read along the lines of 'The next section contains documentation about xy' . In my opinion, consistency is more important than a specific style on this one.

File: docs/4_how_to_search_for_publishers.md
Comment (Weyaaron) [inline docs/4_how_to_search_for_publishers.md:14]: The end is quite sudden. Maybe include a summary? Some sort of tutorial for getting help? Both of these might be done in separate prs.

**Comment by Weyaaron** [comment]
I have provided a list of suggestions in #290. Overall, I like these changes, these are a huge step forward for this project.

File: docs/1_getting_started.md
Comment (MaxDall) [reply docs/1_getting_started.md:12]: Changed this line a bit so it's still referencing the supported table.

File: docs/4_how_to_search_for_publishers.md
Comment (MaxDall) [reply docs/4_how_to_search_for_publishers.md:14]: I also thought about including this in the third tutorial, but yeah, that's something for another PR I'm fine with leaving it as is.

File: docs/3_how_to_filter_articles.md
Comment (Weyaaron) [reply docs/3_how_to_filter_articles.md:166]: Yeah, I have noticed they are consistent. I suggested updating them into the more neutral style. Instead of using personal pronouns (We, You) a neutral description (The next section is about ...) is perhaps more appropriate for written documentation.

Comment (Weyaaron) [reply docs/3_how_to_filter_articles.md:166]: This can be done in a separate pr, since it's a controversial change.

**Review by Weyaaron — APPROVED** [review]

**Pull request merged**
